src/librustdoc/html/render/mod.rs

```diff
@@ -421,6 +421,12 @@ fn settings(root_path: &str, suffix: &str, theme_names: Vec<String>) -> Result<S
             "Theme preferences",
             vec![
                 Setting::from(("use-system-theme", "Use system theme", true)),
+                Setting::Select {
+                    js_data_name: "preferred-theme",
```

Comment:
How is this different than [`theme`]?

[theme]: https://github.com/rust-lang/rust/blob/31e7f23deae308c8df723065c8ad7bee37a9ca7e/src/librustdoc/html/static/js/storage.js#L130

Reply:
Good catch, thanks! For some reason I got the idea that we were currently using a setting called `preferred-theme`, but you're right that it's just `theme`. Will fix.